views/room.ejs

```diff
@@ -135,37 +380,9 @@
                     }
                 });
                 /* This part is needed to make a responsive sidebar: */
-                sidebarHTML = $('#sidebar').eq(0).clone().wrapAll('<div>').parent().html();
-                $('#sidebar').mmenu();
-                if($( document ).width() >= 850){
-                    $('#replaceHolder').html(sidebarHTML);
-                }
-                $( window ).resize(function() {
-                    $("#sidebar").trigger("close.mm");
-                    if($( document ).width() < 850){
-                        $('#replaceHolder').html('');
-                    }else{
-                       $('#replaceHolder').html(sidebarHTML);
-                    }
-                });
+                
                 $('#message-send-button').click(function(){$('form').submit();});
-            });
-            function changeUsername(){
-                var oldUsername = currentUser;
-                var returnedUsername = prompt("Your current username is " + currentUser + ".\nType in a new username:");
-                if(!verifyEmptyness(returnedUsername) && returnedUsername !== null){
-                    localStorage.setItem('MTChat_Username', returnedUsername);
-                    currentUser = returnedUsername;
-                    $(".username").html(nameMessageHTMLPatternBegin + currentUser);
-                    socket.emit('postName', currentUser);
-                }
-            }
-            //Thanks to Jano González from SE for this
-            var verifyEmptyness = function(str) {
```

Comment:
It isn't i changed the function on a other place. verifyEmptyness is also not deleted